spec/factories/items.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :item do
+    title "MyString"
+    description "MyText"
+    price_in_cents 1
+    image_url "MyText"
```

Comment:
I think lines 3 and 6 are going to give us fits since both the title and url need to be unique in the database.   Maybe we start to pull in Faker for something like...

title { Faker::Pokemon.name + "_" + Faker::Number.hexadecimal(3) } # => Pikachu_1f4